commit 624573d0d480a003c9ba042f524f311aef2e7c5f
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:25:58 2010 -0700

    Staging: brcm80211: remove some unused macros
    
    These are all defined by the core kernel, use those versions
    instead of just creating noops.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/linuxver.h
+++ b/drivers/staging/brcm80211/include/linuxver.h
@@ -39,24 +39,6 @@ typedef irqreturn_t(*FN_ISR) (int irq, v
 #include <linux/sched.h>
 #include <linux/ieee80211.h>
 
-#ifndef __exit
-#define __exit
-#endif
-#ifndef __devexit
-#define __devexit
-#endif
-#ifndef __devinit
-#define __devinit	__init
-#endif
-#ifndef __devinitdata
-#define __devinitdata
-#endif
-#ifndef __devexit_p
-#define __devexit_p(x)	x
-#endif
-
-#define netif_down(dev)
-
 /* suspend args */
 #define DRV_SUSPEND_STATE_TYPE pm_message_t
 
